Docs/development/building.md

```diff
@@ -0,0 +1,104 @@
+
+
+#Building
+
+Santa makes use of [rake](https://ruby.github.io/rake/) for building and testing Santa. All of the [releases](https://github.com/google/santa/releases) are made using this same process. Santa's releases are codesigned with Google's KEXT signing certificate. This allows Santa to be loaded with SIP fully enabled. macOS 10.13+ will most likely require user consent, even when signed with Google's KEXT signing certificate.
+
+#### Cloning
+
+Clone the source and change into the directory.
+
+```sh
+git clone https://github.com/google/santa
+cd santa
+```
+
+The above command will default to using the `master` branch. If you wanted to build, run or test a specific version of Santa use this command.
+
+```sh
+git checkout <version, i.e. 0.9.19>
+```
+
+#### Building
+
+Build a debug version of Santa. This keeps all the debug symbols, adds additional logs and does not optimize the compiled output. For speed sensitive tests make sure to benchmark a release version too.
+
+```sh
+rake build:debug
+```
+
+Build a release version of Santa.
+
+```sh
+rake build:release
+```
+
+Both of these just output the binaries that makeup Santa in the default Xcode build location. To actually run  what was built, see the next section.
+
+#### Running
+
+On macOS 10.11+ System Integrity Protection (SIP) prevents loading of kernel extensions that are not signed by an Apple KEXT signing certificate. To be able to load and test a non-release version of Santa, SIP will have to configured to allow non Apple KEXT signing certificates.
```

Comment:
missing "be" and hyphen:
.. SIP will have to be configured to allow non-Apple KEXT signing certificates.